docs/resources/dms_group.md

```diff
@@ -1,11 +1,13 @@
 ---
-subcategory: "Distributed Message Service (DMS)"
+subcategory: "Deprecated"
 ---
 
 # huaweicloud_dms_group
 
 Manages a DMS group in the huaweicloud DMS Service.
 
+-> **NOTE:** Distributed Message Service (Shared Edition) has withdrawn. Please use DMS for Kafka instead.
```

Comment:
suggest to use the exact resource name.

Reply:
Kafka does not have the features of group and queue, so it is impossible to specify exact resource name